src/pages/experiments/[experimentId]/data-exploration/components/embedding/Embedding.jsx

```diff
@@ -109,13 +107,13 @@ const Embedding = (props) => {
   }, [focusedExpression]);
 
   const updateCellCoordinates = (newView) => {
-    if (selectedCell && newView.viewport.project) {
-      const [x, y] = newView.viewport.project([data[selectedCell][0], data[selectedCell][1]]);
+    if (selectedCell && newView.project) {
+      const [x, y] = newView.project(selectedCell);
```

Comment:
am I understanding this correctly that somehow there is a project method in the newView that is aware of the concept of a cell ID?

Reply:
somehow yes. No idea how it works, but when I put the cellId there, it just worked.